app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ArticlesController < ApplicationController
 
-before_action :authenticate_user!, only: [:destroy, :delete, :new, :create, :edit, :update]
+	before_action :authenticate_user!, only: [:destroy, :new, :create, :edit, :update]
```

Comment:
Line is too long. [83/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.